Adding support to building coreclr unix tests in Linux

According to below document(unix test instructions), we can not create the test sets in Linux. It only works on the Windows machine.
(https://github.com/dotnet/coreclr/blob/master/Documentation/building/unix-test-instructions.md)
If there is no logical issue to create unix test in Linux, I want to add buildtest.sh file.
It will be helpful for Linux users.

Please give me informations why linux machine has not supported to create unix test, if there is a special issue.

I created buildtest.sh file for buiding unixtest on Linux machine.
But I faced some problem. 
First, as I know, Roslyn does not support writing PDBs on Unix. So I need to disable debugsymbols on csproj file. We have a lot of csproj files in test/src directory. Is this right way to fix each csproj file manually ?
Seconds, when I build managed code using msbuild, it occurs error with below messages. These targets is provided by ./Tools/Microsoft.Common.CurrentVersion.targets. But my patch does not include it automatically. What should I do ?
`
The target "ResolveReferences" listed in a BeforeTargets attribute at "/home/jyoung/git/dotnet/coreclr/tests/override.targets (20,7)" does not exist in the project, and will be ignored.
The target "BeforeResolveReferences" listed in a BeforeTargets attribute at "/home/jyoung/git/dotnet/coreclr/tests/src/dir.targets (136,11)" does not exist in the project, and will be ignored.
The target "BeforeClean" listed in a BeforeTargets attribute at "/home/jyoung/git/dotnet/coreclr/tests/src/dir.targets (136,11)" does not exist in the project, and will be ignored.
The target "_CopyFilesMarkedCopyLocal" listed in a BeforeTargets attribute at "/home/jyoung/git/dotnet/coreclr/tests/src/dir.targets (163,11)" does not exist in the project, and will be ignored.
`
The last, Desktop.Coreclr.TestWrapperr.csproj does not find System and mscorlib dll. But this code works well in Windows, so I think that I missed some configuration. Please give me a hint.
`
/home/jyoung/git/dotnet/coreclr/tests/src/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs(6,7): error CS0246: The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?) [/home/jyoung/git/dotnet/coreclr/tests/src/Common/Desktop.Coreclr.TestWrapper/Desktop.Coreclr.TestWrapper.csproj]
`
I added my patch and log file.
[Fix_4437.zip](https://github.com/dotnet/coreclr/files/235760/Fix_4437.zip)

--- a/tests/override.targets
+++ b/tests/override.targets
@@ -3,8 +3,6 @@
     Overrides for all other targets (including build tools) can go in this file.
   -->
 
-  <Import Project="mono.targets" Condition="'$(OsEnvironment)'=='Unix'" />
-  <Import Project="roslyn.xplat.targets" Condition="'$(OsEnvironment)'=='Unix'" />
   <!-- Contains overrides for the nuget reference resolution.  The regular nuget reference resolution will not 
        copy references local, which we need in order to correctly execute the xunit project -->
   <Import Project="xunitwrapper.targets" Condition="'$(IsXunitWrapperProject)'=='true'" />

